docs/runtime_environment.rst

```diff
@@ -56,3 +56,11 @@ Note that non-relative imports to these files will `not` work:
 
    import backend
    from utils import img_utils, ml_utils
+
+Limiting GPU memory Growth
+-----------------------------------------
+
+By default Vcap maps all available memory of all visible CUDA configured GPUs,
+To prevent this set Environment varialbe ``TF_FORCE_GPU_ALLOW_GROWTH==True``.
```

Comment:
When setting an environment variable, you only need to use one equals sign. So it would look like `TF_FORCE_GPU_ALLOW_GROWTH=true`. I would also use a lowercase "T" in "true" since that's what the TensorFlow docs recommend.